.github/workflows/test.yaml

```
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - uses: ./.github/actions/install
+      - uses: ./.github/actions/install-v16
       - run: yarn test
```

```suggestion
      - uses: actions/setup-node@v1
        with:
          node-version: 16      
      - run: yarn test
```
Ça ne suffisait pas ? On veut surtout utiliser node v16 pour lancer les tests, il ne devrait pas y avoir d'impact sur l'installation des packages.

---

ca risque pas de casser si on install avec la v14 et qu'on lance la v16 ?

---

Je ne pense pas, les paquets installés ne seront pas différents. À tester, mais je suis plutôt confiant :)

---

j'ai fait la modif, ca a l'air d'être ok !